### PR TITLE
Shelter map popup templating and styles

### PR DESCRIPTION
**Pull request by rdev0rigin**
This PR will provide the `_includes/shelter_map.html` with a similar templating and style for the popups as was implemented with `_includes/map.html`.

screen shot 2018-10-08 at 2 34 19 pm

## concern

- Overlaps (two things) 
    - The popup can be over lapped by the filter radio list. 
    - The map width/height takes up the whole screen making it difficult to scroll to the top or bottom of the view. 

screen shot 2018-10-08 at 2 05 55 pm 
  
### suggestion

- side toolbar - If we have a toolbar on the side that has the filter options it could help with scrolling and be out of the way for popups.  

## concern

- More Details - There seems to be more details that we should want to show, like the `special needs` and `accepting`.

### suggestion

- Details View  - Allowing a separate view with the details might be less confusing for the user.

- Slide-Out - A fixed panel at the bottom that can slide up and down might be a good way to provide scrollable content instead of or with a minimal popup.

### CODE REVIEW

**Comment by rdev0rigin** [comment]
Theres a problem with the controls on mobile views. 
screen shot 2018-10-08 at 5 02 42 pm

**Comment by miklb** [comment]
Good thing about this is that the storm isn't active and this site isn't being used for live data so we can use it as a staging/test environment at this point. A better UI is definitely a priority before it can be used in a live environment.

**Comment by rdev0rigin** [comment]
I refactored the style for mobile for the filter controls.
screen shot 2018-10-08 at 6 56 17 pm
screen shot 2018-10-08 at 6 56 06 pm
screen shot 2018-10-08 at 6 56 29 pm

**Note** I still need to put in the logic for the filters.

File: _includes/shelter_map.html [hunk 1]
```diff
@@ -1,125 +1,332 @@
...
+        zoom: 6.1
+    });
+    var oReq = new XMLHttpRequest();
+    oReq.onload = function (e) {
+        if (!(e && e.target && e.target.response && e.target.response.shelters))
+            return console.error('error fetching shelter JSON');
+        window.shelters = sheltersShown = e.target.response.shelters;
+        populateMarkers(sheltersShown);
+    };
+    oReq.open('GET', 'https://irma-api.herokuapp.com/api/v1/shelters', true);
```
Comment (omnilord) [inline _includes/shelter_map.html]: Putting a comment here so it's easy to find, the real API end point will be at hurricane-florence-api.herokuapp.com.

**Comment by miklb** [comment]
I'm getting an error in the console `ReferenceError: Can't find variable: luxon`

**Comment by rdev0rigin** [comment]
> I'm getting an error in the console `ReferenceError: Can't find variable: luxon`

The latest push should fix it, I am getting an error on the Jekyll server though. 
`[2018-10-09 08:11:20] ERROR `/assets/vendor/luxon.js.map' not found.`

Im not sure why it's telling me this. Looking in to it.

**Comment by rdev0rigin** [comment]
It looks like a Luxon bug and the builds failing. Im going to swap it out for a MomentJS CDN

**Comment by rdev0rigin** [comment]
@miklb Should be fixed now.

**Comment by miklb** [comment]
I'm pulling this down now, but quick question? How much of momentJS are we using? Could something like this https://date-fns.org be lighter to load and still get what we need?

**Comment by rdev0rigin** [comment]
> I'm pulling this down now, but quick question? How much of momentJS are we using? Could something like this https://date-fns.org be lighter to load and still get what we need?

It looks like we can use that, Ill plunk it in.

**Comment by miklb** [comment]
As long as it can serve the same purpose and be loaded modularly and thus lighter. We can always refactor later if it’s a bigger task.

**Comment by rdev0rigin** [comment]
@miklb Good idea with date-fns. Its in and working.

**Comment by rdev0rigin** [comment]
I added the location finder that comes out the (map)box and removed the `NearBy` control.
screen shot 2018-10-09 at 12 17 56 pm

**Pull request merged**